instagram-app/src/components/PostContainer/CommentSection/CommentSectionStyles.js

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components'
+
+const CommentSectionDiv = styled.div`
```

Comment:
I would recommend naming your styled components `StyledCommentDiv` for example, just to be clear as to which are regular components and which are styled.